iac/terraform/aws/modules.tf

```diff
@@ -2,3 +2,23 @@ module "auth" {
   source      = "./auth"
   environment = var.environment
 }
+
+module "database" {
+  source     = "./dynamodb"
+  table_name = var.table_name
+}
+
+module "domain_verification" {
+  source      = "./domainVerification"
+  domain_name = var.domain_name
+}
+
+module "cognito" {
+  source              = "./auth_cognito"
```

Comment:
How about only `cognito` instead of `auth_cognito`

Reply:
At first it was just cognito but later changed to auth_cognito but now I change to cognito. [commit](https://github.com/CraftsmenLtd/BloodConnect/commit/28f045306270fdc86a03546bd3b3e08356c4dc74#diff-97cf88c848d441b818c42a0a69a22b4659c1c58abbe6712c04a758c7bc15ba9b)